### PR TITLE
KAA-1077 Add patch which disables tests for avro

### PR DESCRIPTION
**Pull request by ohamov**

### CODE REVIEW

**Comment by forGGe** [comment]
Typo: disaples -> disables

**Comment by forGGe** [comment]
+

File: client/client-multi/client-cpp/tools/avro-cpp-disable-tests.patch [hunk 1]
```diff
@@ -0,0 +1,101 @@
+diff -crB avro-cpp-1.7.5/rpi.cmake avro-cpp-1.7.5_patched/rpi.cmake
+*** avro-cpp-1.7.5/rpi.cmake	2015-12-11 12:35:05.013408577 +0200
+***************
+*** 0 ****
+--- 1,4 ----
++ set(CMAKE_SYSTEM_NAME Linux)
++ set(CMAKE_C_COMPILER $ENV{CROSS}-gcc)
++ set(CMAKE_CXX_COMPILER $ENV{CROSS}-c++)
++ set(CMAKE_FIND_ROOT_PATH  $ENV{INSTALLDIR}/arm-linux-gnueabihf/libc/usr)
```
Comment (rasendubi) [inline client/client-multi/client-cpp/tools/avro-cpp-disable-tests.patch]: This block doesn't disable tests, so maybe rename patch or split it.

File: client/client-multi/client-cpp/pom.xml [hunk 2]
```diff
@@ -79,6 +79,7 @@
                         <exclude>**/.cproject</exclude>
                         <exclude>**/kaa_status.file</exclude>
                         <exclude>**/FindSqlite3.cmake</exclude>
+                        <exclude>**/tools/avro-cpp-disable-tests.patch</exclude>
```
Comment (rasendubi) [inline client/client-multi/client-cpp/pom.xml]: Don't use globbing.

**Comment by rasendubi** [comment]
LGTM

**Pull request merged**